pyproject.toml

```diff
@@ -6,9 +6,9 @@ description = "Chia blockchain full node, farmer, timelord, and wallet."
 authors = ["Mariano Sorgente <mariano@chia.net>"]
 license = "Apache License"
 readme = "README.md"
-keywords= ["chia", "blockchain", "node"]
+keywords = ["chia", "blockchain", "node"]
```

Comment:
is the reformatting throught this file intentionally being introduced by ruff?  or...  prettier is supposed to be processing toml so i'm a bit annoyed that this is was apparently so messy.

Reply:
You know, I'm not sure.  There were a few stages of getting this PR ready locally.  One iteration ran the formatter and it's possible out of a desire to keep the changes to this file, I accidentally kept all of the formatting as well.  That's my best guess, but I can confirm if you like.  The formatting seems good as you noted, so I don't see a harm in keeping it around.

Reply:
Confirmed that this does not get re-added if I remove some of it.  But like I said, I think we should keep it.

Reply:
i think we're just missing a prettier plugin for handling toml files.  i'm working on adding that.  would you be willing to narrow this down to just the 'real' changes and i'll swing back through afterwards with a pr where prettier cleans it up?  assuming i really get it working.